libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-dialog.component.ts

```diff
@@ -0,0 +1,165 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormlyFieldConfig,FormlyFormOptions, Field } from '@ngx-formly/core';
+import { BehaviorSubject } from 'rxjs';
+import {
+    SdsDialogService,
+    SdsDialogRef,
+    SDS_DIALOG_DATA
+  } from '@gsa-sam/components';
+import { removeSummaryDuplicates } from '@angular/compiler';
+import { text } from '@angular/core/src/render3';
+import { _MatButtonMixinBase } from '@angular/material/button';
+  export interface DialogData {
+    keyword: boolean;
```

Comment:
what is this data for..?

Reply:
This is not required anymore...I have made the changes and removed it